object-manipulation-1/src/get-value.js

```diff
@@ -0,0 +1,9 @@
+/* exported getValue */
+function getValue(object, key) {
+  for (var key2 in object) {
+    if (object[key] === object[key2]) {
+      return object[key];
+    }
+  }
+  return undefined;
```

Comment:
This function shouldn't require a loop. You have a parameter for the object and a parameter for the property name, you should just access that value and return it